lib/expander/src/Events/CourseModuleCompleted.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nikolay.mikov
+ * Date: 5.6.2017 г.
+ * Time: 11:32
+ */
+
+namespace LogExpander\Events;
+
+
+class CourseModuleCompleted extends  Event
+{
+    public function read(array $opts)
+    {
+
+        global $DB;
```

Comment:
@nikidev if you make new commits to this branch they will be reflected in this PR